### PR TITLE
modified readme

### PR DESCRIPTION
**Pull request by harshagr70**
tried adding some more data

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,4 +25,8 @@ We have a variety of issues available for contributors to work on, including bug
 -Develop a resource page for hackathon organizers, including a list of tools and resources for planning, promoting, and running successful hackathons.
 
 
-Thank you for your interest in OpenDevFellows, and we look forward to collaborating with you!
\ No newline at end of file
+Thank you for your interest in OpenDevFellows, and we look forward to collaborating with you!
+
+# support
```
Comment (AbhiiVops) [inline README.md:30]: Try making it more catchy and add it above the license
In the support section of the readme for the OpenDevFellows repository, you can provide the following information:

1. Contact Information: Provide an email address or a link to a contact form where users can ask questions or report issues related to the repository.
Add the link : https://linktr.ee/theonlyabhii
3. Contributing Guidelines: Link to the contributing guidelines for the repository to encourage users to contribute to the project.

**Review by AbhiiVops — CHANGES_REQUESTED** [review]
Please check the review and make your repo up to date with the upstream changes . You can do this by :
`git remote add upstream <upstream-repository-url>`
`git fetch upstream`
`git checkout <your-branch>` (if created a separate branch)
`git merge upstream/<branch-to-merge>`
`git push origin <your-branch>`

**Comment by AbhiiVops** [comment]
> tried adding some more data

Get it assigned first the next time you contribute

File: README.md (hunk 1 above)
Comment (harshagr70) [reply README.md:30]: sure ... i ll work upon that once again ,,,

**Comment by harshagr70** [comment]
> > tried adding some more data
> 
> Get it assigned first the next time you contribute

i ll make this sure before my next pr

**Pull request closed without merging**